Sources/Guaka/Flag/Flag.swift

```diff
@@ -56,14 +56,31 @@ public struct Flag: Hashable {
   public let type: FlagValue.Type
 
 
-  /// Set the flag to be required/
+  /// Set the flag to be required
   /// A required flag must be set in order for commands to be executed.
   public let required: Bool
 
 
+  /// Flag values.
+  /// The value can be `int`, `bool`, `string` or any other type that implements `FlagValue`
+  public var values: [FlagValue] = []
+
+
+  /// Set the flag to allow multiple values
+  /// A repeatable flag can be specified multiple times and will return an array of values.
+  public var repeatable: Bool
+
+
   /// Flag value.
   /// The value can be `int`, `bool`, `string` or any other type that implements `FlagValue`
-  public var value: FlagValue?
+  public var value: FlagValue? {
+    get { return values.first }
+    set {
+      if let val = newValue {
+        values = [val]
+      }
+    }
+  }
```

Comment:
I do this to prevent breaking current code where users may be manually setting a default value for one of their non-required flags.